platform/tools/native-call.js

```diff
@@ -4,8 +4,16 @@ const base = require('../core/base');
 
 //TODO: also there are some issues with error handling here.
 //
-const nativeCall = (path, inputs, callback, error, context) => new Promise((resolve, reject) => {
-    core.callable(() => new core.Call(path), context)(inputs).then(result => {
+const nativeCall = (pathObject, inputs, callback, error, context) => new Promise((resolve, reject) => {
+    let path = pathObject;
+    let method = 'get';
+
+    if(path instanceof Object) {
+      path = pathObject.path;
+      method = pathObject.method;
```

Comment:
what if `pathObject.method` is `undefined`?

Reply:
Fixed this to check for its existence.